ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -42,7 +43,8 @@ internal constructor(
   private val navigator: Navigator,
   private val offlineAreaRepository: OfflineAreaRepository,
   private val submissionRepository: SubmissionRepository,
-  private val surveyRepository: SurveyRepository,
+  val surveyRepository: SurveyRepository,
+  val userRepository: UserRepository,
```

Comment:
Let's keep these as private

Reply:
These are getting used in HomeScreenFragment, so we cannot make them private